ElastosBountyProgram/front-end/src/module/form/UserEditForm/Component.js

```diff
@@ -116,6 +116,23 @@ class C extends BaseComponent {
         const username_el = (
             <Input size="large" disabled/>
         )
+        
+        const role_fn = getFieldDecorator('role', {
+            rules: [{required: true, message: 'Please select your role'}],
```

Comment:
1. Please select a role (not yours, the admin potentially edits someone elses account)
2. Please move that string to I18N